n2y/notion.py

```diff
@@ -373,3 +373,18 @@ def save_file(self, content, page, extension):
         with open(full_filepath, 'wb') as temp_file:
             temp_file.write(content)
         return urljoin(self.media_url, relative_filepath)
+
+    def append_block_children(self, block_id, children):
+        response = requests.patch(
+            f"{self.base_url}blocks/{block_id}/children",
+            json={"children": children}, headers=self.headers
+        )
+        return self._parse_response(response)
+
+    def delete_block(self, block_id):
+        headers = self.headers
+        del headers['Content-Type']
```

Comment:
I think that deleting this here will delete it everywhere that else that uses `self.headers`, thus you may need to copy `self.headers` first or find some other way to remove Content-Type